application/backend/app/api/endpoints/projects.py

```diff
@@ -125,6 +125,27 @@ def get_project(
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=str(e))
 
 
+@router.patch(
+    "/{project_id}",
+    response_model=Project,
+    responses={
+        status.HTTP_200_OK: {"description": "Project name updated successfully"},
+        status.HTTP_400_BAD_REQUEST: {"description": "Invalid project ID or request body"},
+        status.HTTP_404_NOT_FOUND: {"description": "Project not found"},
+    },
+)
+def update_project(
+    project_id: Annotated[UUID, Depends(get_project_id)],
+    project_update: Annotated[ProjectUpdate, Body(description="Updated project name")],
+    project_service: Annotated[ProjectService, Depends(get_project_service)],
+) -> Project:
+    """Update project name only"""
```

Comment:
```suggestion
def rename_project(
    project_id: Annotated[UUID, Depends(get_project_id)],
    project_update: Annotated[ProjectUpdate, Body(description="Updated project name")],
    project_service: Annotated[ProjectService, Depends(get_project_service)],
) -> Project:
    """Rename a project"""
```
^ Minor adjustments to improve the auto-generated API spec